WordPress/Classes/Services/SuggestionService.swift

```diff
@@ -32,16 +41,18 @@ class SuggestionService {
     */
     private func fetchAndPersistSuggestions(for blog: Blog, completion: @escaping ([UserSuggestion]?) -> Void) {
 
+        guard let blogId = blog.dotComID, let hostname = blog.hostname else { return }
```

Comment:
```suggestion
        guard let blogId = blog.dotComID, let _ = blog.hostname else { return }
```

Reply:
Thanks for catching! This ended up being an unused variable I'd forgotten to delete 🤦